FetchMigration/README.md

```diff
@@ -1,74 +1,76 @@
+source cluster, such as Elasticsearch or OpenSearch, to a target OpenSearch cluster. It automates the process of 
+comparing indexes between the two clusters by creating index metadata, like settings and mappings that does not already exist on the target cluster using [Data Prepper](https://github.com/opensearch-project/data-prepper).
 
-The Fetch Migration tool is implemented in Python.
-A Docker image can be built using the included [Dockerfile](./Dockerfile).
+The Fetch Migration tool consists of a Python script which requires Python 3 or greater. However, a Docker image can be built using the included [Dockerfile](./Dockerfile) included in this repository.
 
 ## Components
 
-The tool consists of 3 components:
-* A "metadata migration" module that handles metadata comparison between the source and target clusters. 
-This can output a human-readable report as well as a Data Prepper pipeline `yaml` file.
-* A "migration monitor" module that monitors the progress of the migration and shuts down the Data Prepper pipeline 
+The tool consists of three components:
+
+* The **Metadata migration** module handles metadata comparison between the source and target clusters. This can output a human-readable report as well as a Data Prepper `pipeline.yaml` file.
+* The **Migration monitor** module monitors the progress of the migration and shuts down the Data Prepper pipeline 
 once the target document count has been reached
-* An "orchestrator" module that sequences these steps as a workflow and manages the kick-off of the Data Prepper 
-process between them.
+* The **Orchestrator** module that sequences these steps as a workflow and manages the kick-off of the Data Prepper 
+process between them. The orchestrator module helps route the correct components when using Docker, though each component can be executed seperately using Python.
 
-The orchestrator module is the Docker entrypoint for the tool, though each component can be executed separately 
-via Python. Help text for each module can be printed by supplying the `-h / --help` flag.
+Help text for each module can be printed using the `-h / --help` flag when exectuing the Python script.
 
 ## Current Limitations
 
-* Fetch Migration runs as a single instance and does not support vertical scaling or data slicing
-* The tool does not support customizing the list of indices included for migration
-* Metadata migration only supports basic auth
-* The migration does not filter out `red` indices
-* In the event that the migration fails or the process dies, the created indices on the target cluster are not rolled back
+The Fetch Migration currently has the following limitations:
+
+* Fetch Migration runs as a single instance and does not support vertical scaling or data slicing.
+* Fetch Migration does not support customizing the list of indices included for migration.
+* Fetch migration only supports basic authentication.
```

Comment:
Note - this line should be removed once https://github.com/opensearch-project/opensearch-migrations/pull/394 is merged